src/Gui/Selection/Selection.cpp

```diff
@@ -391,6 +393,19 @@ void SelectionSingleton::enablePickedList(bool enable)
     }
 }
 
+static void notifyDocumentObjectViewProvider(const SelectionChanges& changes) {
+    const auto* doc = App::GetApplication().getDocument(changes.pDocName);
+    if (!doc) return;

```

Comment:
our code style requires `{ }` even for one liners
```suggestion
    if (!doc) {
        return;
    }
```